framework/generated/khronos_generators/vulkan_generators/layer_func_table_generator.py

```diff
@@ -126,17 +138,3 @@ def need_feature_generation(self):
         if self.feature_cmd_params:
             return True
         return False
-
-    def generate_feature(self):
```

Comment:
Easier to review if you simply rename the function/variables in situ, and then call from endFile